components/eam/src/physics/cam/physics_utils.F90

```diff
@@ -1,88 +1,88 @@
 ! physics_utils: common elements for SCREAM physics models.
 module physics_utils
-
+  
```

Comment:
@singhbalwinder , did you add all of these extra spaces for a reason?  It seems to me this may put EAMxx in conflict with upstream E3SM for no real reason.

Reply:
I was working on Cori with a different editor which added these white spaces. I will fix these in my next commit.